docs/source/howto/calculations/pw.rst

```diff
@@ -311,3 +311,38 @@ To retrieve additional output files, specify the list of files in the ``ADDITION
 
 .. |SSSP| replace:: Standard Solid-State Pseudopotentials (SSSP)
 .. _SSSP: https://www.materialscloud.org/discover/sssp/table/efficiency
+
+
+How to analyze the results
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a ``PwCalculation`` is completed, there are quite a few possible analyses to perform.
+
+How to check the SCF accuracy during the self-consistent cycle
+--------------------------------------------------------------
+
+During the self-consistent field cycle, the difference in energy of the newly computed charge density and the starting one is computed and stored.
+It can easily be retrieved through the :meth:`aiida_quantumespresso.tools.calculations.pw.PwCalculationTools.get_scf_accuracy` method.
```

Comment:
Can we somehow not show the entire path to the method?

<img width="770" alt="Screenshot 2023-05-09 at 06 00 32" src="https://user-images.githubusercontent.com/25607960/236990731-c706d98e-c9ff-4813-9506-dfd2577625ac.png">

Reply:
yeah we can, just put `~` in front of it.